01-Login/exec.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+docker build -t auth0-spring-security-mvc-webapp .
```

Comment:
Would be nice to tag them under a "samples" category with a "latest" version tag. I was using the next one:

`auth0-samples/{REPO}-{PATH}:latest` -> `auth0-samples/spring-mvc-00login:latest`